Firestore/core/test/firebase/firestore/testutil/async_testing.cc

```diff
@@ -36,6 +40,62 @@ std::shared_ptr<util::AsyncQueue> AsyncQueueForTesting() {
   return std::make_shared<AsyncQueue>(ExecutorForTesting("worker"));
 }
 
+// MARK: - Expectation
+
+Expectation::Expectation() : promise_(std::make_shared<std::promise<void>>()) {
```

Comment:
Note: you could move this into an in-class initializer and have an `= default` constructor.

Reply:
Note: this is unchanged. Once again, it's a purely optional suggestion -- just making sure this is intentional.